controllers/users.js

```diff
@@ -135,10 +145,56 @@ const updateAvatar = async (req, res, next) => {
   res.redirect(`/avatars/${fileName}`);
 };
 
+const userVerification = async (req, res, next) => {
+  try {
+    const { verificationToken } = req.params;
+    const user = await User.findOne({ verificationToken });
+    if (!user) {
+      return res.status(404).json({ message: "User not found" });
+    }
+    user.verify = true;
+    user.verificationToken = null;
+    console.log(user);
+
+    await user.save();
+
+    res.status(200).json({ message: "Verification successful" });
+  } catch (err) {
+    next(err);
+  }
+};
+
+const userReVerification = async (req, res, next) => {
+  try {
+    const email = req.body;
+    if (!email) {
+      return res.status(400).json({ message: "missing required field email" });
+    }
+    const user = await User.findOne(email);
+
+    if (user.verify === true) {
```

Comment:
```suggestion
    if (user.verify) {
```
Inny sposób zapisu :)